### DISCUSSION TITLE
Double click on file launches app but never opend ther file

### DISCUSSION THREAD

**Discussion by ufopsi** [samschott/maestral#887] [Q&A]
Hi, I recently found a bug with 1.7.1: when I double click on a file (such as a PDF, Excel, ...), the app opens but the document never opens. Same if I select "Open with..." from contextual menu. I have to go to the app and there open my file.

This happens across 2 machines. 
Documents outside Dropbox behave normally. 
Tried rebuilding the index.

What can I do?
Tnx
Regsrds.
M

**Comment by arfgithub** [comment]
Hi @ufopsi Same thing with me, just opened here (GitHub) an [issue](https://github.com/samschott/maestral/issues/949). Also, after synching file does not open to from Dropbox web.
